### PR TITLE
Feature/did set

### PR DESCRIPTION
**Pull request by lucblender**
Fixed @offline bug
at @set message, @update were sent -> now @didSet are send

### CODE REVIEW

**Comment by lucblender** [comment]
I did change the persistence db file name in 6dcf28b making it uuid-persistence.db so we can create multiple endpoints in the same code.

File: src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java [hunk 1]
```diff
@@ -480,7 +480,6 @@ public void rollbackTransaction(){
         private static final String SSL_VERIFY_HOSTNAME_DEFAULT     = "true";
 
         /*** MapDB parameters******************************************************************************************/
-        private static final String PERSISTENCE_FILE                = "cloudiOPersistenceData.db";
```
Comment (cm0x4D) [inline src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java:483]: 👌

File: src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java [hunk 2]
```diff
@@ -1044,7 +1048,37 @@ private void set(String topic, Stack<String> location, CloudioMessageFormat mess
                     CloudioAttribute.InternalAttribute attribute = node.findAttribute(location);
                     if (attribute != null) {
                         // Deserialize the message into the attribute.
-                        messageFormat.deserializeAttribute(data, attribute);
+                        String correlationID = messageFormat.deserializeSetAttribute(data, attribute);
+                        System.out.println(correlationID);
```
Comment (cm0x4D) [inline src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java]: Remove debug output

File: src/main/java/ch/hevs/cloudio/endpoint/JsonMessageFormat.java [hunk 3]
```diff
@@ -130,6 +130,32 @@
         return outputStream.toByteArray();
     }
 
+    @Override
+    public byte[] serializeDidSetAttribute(CloudioAttribute.InternalAttribute attribute, String correlationID) {
```
Comment (cm0x4D) [inline src/main/java/ch/hevs/cloudio/endpoint/JsonMessageFormat.java:134]: Add timestamp for the case the message can not be delivered instantaneous.

**Review by cm0x4D — CHANGES_REQUESTED** [review]

File: src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java (hunk 2 above)
Comment (lucblender) [reply src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java]: Oups, someone probably took my computer to add ugly println

**Review by cm0x4D — APPROVED** [review]

**Pull request merged**